doc/services/mem_mgmt/index.rst

```diff
@@ -0,0 +1,90 @@
+.. _mem_mgmt_api:
+
+Memory Attributes
+#################
+
+It is possible in the DeviceTree to mark the memory regions with attributes by
```

Comment:
nonblocking: 'devicetree' (or 'Devicetree' at beginning of sentence) is what we do, for consistency with the DT spec. This applies elsewhere in the patch too